docs/machine-learning/how-to-guides/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ML.NET How-to guides
+description: Learn how to do specific tasks to assist with custom AI solutions creation and Machine Learning integration into your .NET applications.
+ms.date: 10/05/2018
+---
+# How to (ML.NET)
+
+In the How to section of the ML.NET Guide you can find quick answers
+to common questions. In some cases, articles may
+be listed in multiple sections. We wanted to make them easy to find
```

Comment:
can you rewrite this without using we?

Maybe:
In some cases, articles may be listed in multiple sections to make them easy to find...